fetch.bs

```diff
@@ -3061,6 +3251,31 @@ in addition to <a>HTTP fetch</a> above.
   nullity has already been checked. The <a lt=extract for=BodyInit>extracting</a> operation cannot
   throw as it was called for the same <a for=body>source</a> before.
 
+ <li><p>Let <var>useReferredTokenBinding</var> be the result of
+   <a for=header>parsing</a> `<code>Include-Referred-Token-Binding-ID</code>`
+   in <var>actualResponse</var>'s <a for=response>header list</a>.
+
+ <li><p>Unset the <var>request</var>'s <a for=request>use-referred-token-binding flag</a>.
+
+ <li><p>If <var>useReferredTokenBinding</var> is a <span>byte-case-insensitive</span>
+     match for `<code>true</code>`, and the <var>request</var>'s
+     <a for=request>use-token-binding flag</a> is set, then perform the following substeps:
```

Comment:
Style nit: Inconsistent wrapping here and above. Please follow the example of the rest of the steps in these algorithms.